src/modules/Dropdown/Dropdown.js

```diff
@@ -108,13 +108,13 @@ export default class Dropdown extends Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
-    this.element.dropdown('refresh')
+    _.invoke(this, 'element.dropdown', 'refresh')
```

Comment:
Ignore all dropdown work, this is completely rewritten in #206.  These changes only made to pass CI.